content/en/docs/ops/deployment/deployment-models/index.md

```diff
@@ -29,15 +29,18 @@ or are there multiple control planes deployed to ensure high availability (HA)?
 Are all clusters going to be connected into a single {{< gloss >}}multicluster{{< /gloss >}}
 service mesh or will they be federated into a {{< gloss >}}multi-mesh{{< /gloss >}} deployment?
 
-All of these questions, among others, represent independent dimensions of configuration for an Istio deployment.
+## Definition of a Mesh in Istio
+
+Before answering the questions above, it is imperative to understand the definition of a `mesh` in Istio.
+
+_A mesh is one single administrative domain, consisting of one single shared root CA trusted by 
```

Comment:
A mesh can actually have multiple roots, so long as all clusters trust the same set of roots it is a single mesh.

I'm not even sure how to define a mesh anymore, since it reality Istio is really flexible and even service discovery doesn't have to be the same in every cluster.

But I think administrative domain / shared set of trusted roots / single service discovery / service naming domain is kind of the right direction?

Reply:
Is this the right place for this definition? We already have [What is a Service Mesh](https://istio.io/latest/docs/concepts/what-is-istio/#what-is-a-service-mesh) ... maybe this re-work should go there instead and we could link to it?

Reply:
The `What is a Service Mesh` section of the concepts is very high level, so I don't think we want this there, but I do think we should clean up the [service mesh glossary entry](https://preliminary.istio.io/latest/docs/reference/glossary/#service-mesh) should say this, and here we can refer to and highlight the single admin domain part